public/video-ui/src/constants/TagTypes.js

```diff
@@ -22,4 +22,8 @@ export default class TagTypes {
   static get tone() {
     return 'tone';
   }
+
+  static get paidContent() {
```

Comment:
This is very minor so feel free to ignore but `paidContentKeywords` might be a better name - this type includes more than just paid content tags.